ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_logic.py

```diff
@@ -468,3 +468,31 @@ def test_paddle_isclose(
         y=x[1],
         equal_nan=equal_nan,
     )
+
+
+# logical_and
+@handle_frontend_test(
+    fn_tree="paddle.logical_and",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"), num_arrays=2, shared_dtype=True
+    ),
+    test_with_out=st.just(True),
```

Comment:
No need for this. It's true by default.

Reply:
yeah ,I removed this from here. now its working perfect.